Podfile

```diff
@@ -100,6 +100,12 @@ post_install do |installer|
       create_symlink_phase.always_out_of_date = "1"
     end
 
+    if target.name.start_with? "AlphaWallet"
+      target.build_configurations.each do |config|       
+        config.build_settings['SWIFT_VERSION'] = '5.0'
```

Comment:
Why not set this in the project file instead of overriding it here?

Reply:
Something overrides the settings. I set the swift language version to 5.0 and running make bootstrap resets it to 4.2. Also git does not register any changes that could be saved. @hboon

Reply:
@eviltofu how did you verify this:

> running make bootstrap resets it to 4.2.

Reply:
I change the Swift Language Version to 5.0 of a Pod. Then I go to the command line and type in `make bootstrap`. After `make bootstrap` is done, I go back to Xcode and check the version number.

Before | After
-|-
<img width="1190" alt="before" src="https://github.com/AlphaWallet/alpha-wallet-ios/assets/1050309/8a6dd86f-a621-43ea-a774-ee1494af5eac"> | <img width="1197" alt="after" src="https://github.com/AlphaWallet/alpha-wallet-ios/assets/1050309/eafc9c7b-33bb-40cb-aaa3-7cc30818ed88">

Reply:
Updated all AlphaWallet* podspec files to use swift version 5.0 @hboon